src/app/app-routing.module.ts

```diff
@@ -1,11 +1,23 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
-import { ContentPageComponent } from './components/content-page/content-page.component';
 import { LoginPageComponent } from './components/login-page/login-page.component';
+import { MainPageComponent } from './components/main-page/main-page.component';
+import { ProfilePageComponent } from './components/profile-page/profile-page.component';
+import { HomeFeedComponent } from './components/home-feed/home-feed.component';
 
 const routes: Routes = [{
   path: '',
-  component: ContentPageComponent
+  component: MainPageComponent,
+  children: [
+    {
+      path: '',
```

Comment:
We now are not able to Route to for e.g :      '/profile'  by URL ?

Reply:
We are using this url to go to the profile feed for now. This will change when we have something to show in the home feed.